ynput/OpenPype
# fusion: AYON renaming menu

## Changelog Description
Fusion is having Ayon menu.

## Testing notes:
1. open Fusion in Ayon and see the difference
2. In case the AYON menu is not showing up with correct name in window label, try following
 - remove content of `[HOME]\AppData\Roaming\Blackmagic Design\Fusion`
 - remove content of `[HOME]/.openpype/hosts/fusion`
